main.js

```diff
@@ -4,13 +4,14 @@ const BrowserWindow = electron.BrowserWindow
 const path = require('path')
 const url = require('url')
 const fs = require('fs')
-const autoUpdater = require('electron-updater').autoUpdater
-const autoUpdaterInterval = 60 * 60 * 1000;
 const ipc = electron.ipcMain;
 const Menu = electron.Menu;
 const shell = electron.shell;
 const ElectronConfig = require('electron-config');
 
+const { initializeAutoUpdater } = require('./autoupdate');
+const { getLocaleMessages, normalizeLocaleName } = require('./locale');
```

Comment:
I'd be ok with 

```js
const locale = require("./locale");
locale.getLocaleMessages();
// etc
```

I was just feeling like `main.js` was a giant blob of code and wanted to pull some of it out into focused files.